pkgs/misc/drivers/openzwave/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchFromGitHub
+, doxygen, libxml2, git, which, pkgconfig, graphviz-nox
+, udev }:
+
+let
+  version = "2018-04-04";
+
+in stdenv.mkDerivation rec {
+  name = "openzwave-${version}";
+
+  src = fetchFromGitHub {
+    owner = "OpenZWave";
+    repo = "open-zwave";
+    rev = "ab5fe966fee882bb9e8d78a91db892a60a1863d9";
+    sha256 = "0yby8ygzjn5zp5vhysxaadbzysqanwd2zakz379299qs454pr2h9";
+  };
+
+  nativeBuildInputs = [ doxygen libxml2 git which pkgconfig graphviz-nox ];
+  buildInputs = [ udev ];
+
+  preBuild = ''
+    makeFlagsArray+=("DESTDIR=$out")
+    makeFlagsArray+=("PREFIX=$out")
```

Comment:
Try removing this line and you shouldn't need the fixup anymore.

Reply:
Removing that created a whole range of other issues, because there's a thing that doesn't respect the `DESTDIR`...

So currently it creates:
```
/nix/store/xyz-openzwave-2018-04-04/lib/pkgconfig/libopenzwave.pc
/nix/store/xyz-openzwave-2018-04-04/nix/store/xyz-openzwave-2018-04-04/<rest-of-the-files>
```

I haven't found a good workaround for that :/

Reply:
Ok, I've gone through all the things now.

I've managed to set `PREFIX=` (as in empty string to not get the default `/usr/local/bin`).

And got rid of  most of fixup but had to patch the pkgconfig-file so the python library can be built.

I have tested it with the python library on x86_64 with home assistant as well. So at the moment it works and is slightly less ugly :)